app/src/controllers/TorusKeyring.js

```diff
@@ -1,7 +1,8 @@
 import { EventEmitter } from 'events'
 import Wallet from 'ethereumjs-wallet'
 import * as ethUtil from 'ethereumjs-util'
-import sigUtil from 'eth-sig-util'
+// import sigUtil from 'eth-sig-util'
+const sigUtil = require('eth-sig-util')

```

Comment:
Make it to run with import style... check the eth-sig-util code to see how to import in in latest version

Reply:
Tried, not working